source/jormungandr/tests/journey_common_tests.py

```diff
@@ -368,9 +368,13 @@ def test_journeys_wheelchair_profile(self):
         self.is_valid_journey_response(response, query)
 
         assert(len(response['journeys']) == 2)
+
+        def compare(l1, l2):
```

Comment:
since it's used only there I more for letting it there :stuck_out_tongue_winking_eye: